python/dataproc_templates/util/template_constants.py

```diff
@@ -45,6 +45,12 @@
 FORMAT_HBASE = "org.apache.hadoop.hbase.spark"
 TABLE = "table"
 TEMP_GCS_BUCKET="temporaryGcsBucket"
+MONGO_URL = "spark.mongodb.output.uri"
+MONGO_DATABASE = "database"
+MONGO_COLLECTION = "collection"
+FORMAT_MONGO = "com.mongodb.spark.sql.DefaultSource"
+MONGO_DEFAULT_BATCH_SIZE = 512
```

Comment:
why 512 ? Seems very peculiar number.
A round number like 1000 makes more sense to me, unless there is a reason for it.

Reply:
Actually 512 is the default batch size provided by MongoDB. Hence, I mentioned the same. I have changed it to 1000.